modules/custom/moj_resources/src/Utilities.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
Actually, thinking, do we not want to break this off into it's own PR? Then it can be merged and used by anything using it afterwards 👍

Reply:
We can, or depending on whether we merge to `main` or to a release branch it should be available